source/_integrations/simplisafe.markdown

```diff
@@ -2,6 +2,7 @@
 title: SimpliSafe
 description: Instructions on how to integrate SimpliSafe into Home Assistant.
 ha_release: 0.81
+ha_iot_class: Cloud Poll
```

Comment:
Cloud Polling

Reply:
Doh!

I think I got the update made.  Please confirm.